frontend/server/controllers/CourseController.php

```diff
@@ -317,12 +313,12 @@ public static function apiCreate(Request $r) {
             'name' => $r['name'],
             'description' => $r['description'],
             'alias' => $r['alias'],
-            'school_id' => is_null($r['school']) ? null : $r['school']->school_id,
+            'school_id' => $r['school_id'] ?? null,
```

Comment:
nit:

```suggestion
            'school_id' => $r['school_id'],
```

en general siempre es seguro reemplazar `$x ?? null` por `$x`.

Reply:
listo